app/models/grda_warehouse/hud/affiliation.rb

```diff
@@ -4,6 +4,7 @@ class Affiliation < Base
     self.table_name = 'Affiliation'
     self.hud_key = :AffiliationID
     acts_as_paranoid column: :DateDeleted
+    has_many :users
```

Comment:
@everestwitman is this intentional?  These should not be related to users in any way that I can think of.